### DISCUSSION TITLE
Welcome to Python-Basic-For-All-3.x Discussions!

### DISCUSSION THREAD

**Discussion by reddyprasade** [reddyprasade/Python-Basic-For-All-3.x#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dinesh445** [comment]
Hello sir!
I would like to start project on prediction of fight moves. I hope you watch "Fast & Furious Presents: Hobbs & Shaw" this film, so I'm interested to do project about the ability of Idris Elba having in film. I hope you will guide me. Thank you sir🙂🙂

**Reply by reddyprasade** [reply]
Sure ma

**Reply by reddyprasade** [reply]
Collect the Data Regarding that we will Do that

**Comment by Dinesh445** [comment]
Thank you so much sir for your kind reply. Right now I have almost zero
knowledge on coding . Could you suggest me what I have to learn sir?

**Reply by reddyprasade** [reply]
Sure ma
